### ISSUE TITLE
@JSON has no effect

### ISSUE THREAD

**Issue by F0x1d** [joshuawright11/papyrus#42]
I have this macro:
```swift
extension JSONDecoder {
    static var testing: JSONDecoder {
        Container.shared.jsonDecoder()
    }
}
extension JSONEncoder {
    static var testing: JSONEncoder {
        Container.shared.jsonEncoder()
    }
}

@API
@Mock
@JSON(
    encoder: .testing,
    decoder: .testing
)
@KeyMapping(.snakeCase)
protocol Auth {
    @POST("/register")
    @JSON(
        encoder: .testing,
        decoder: .testing
    )
    func register(body: Body<RegisterRequestBody>) async throws -> TokensResponse
}
```

But in expansion i see:
```swift
struct AuthAPI: Auth {
    private let provider: PapyrusCore.Provider

    init(provider: PapyrusCore.Provider) {
        self.provider = provider
    }

    func register(body: Body<RegisterRequestBody>) async throws -> TokensResponse {
        var req = builder(method: "POST", path: "/register")
        req.requestEncoder = .json(JSONEncoder())
        req.responseDecoder = .json(JSONDecoder())
        req.setBody(body)
        let res = try await provider.request(req)
        try res.validate()
        return try res.decode(TokensResponse.self, using: req.responseDecoder)
    }

    private func builder(method: String, path: String) -> RequestBuilder {
        var req = provider.newBuilder(method: method, path: path)
        req.requestEncoder = .json(JSONEncoder())
        req.responseDecoder = .json(JSONDecoder())
        req.keyMapping = .snakeCase
        return req
    }
}
```

According to 
```swift
req.requestEncoder = .json(JSONEncoder())
req.responseDecoder = .json(JSONDecoder())
```
in both places, It looks like @JSON has no effect on macro

**Comment by F0x1d** [comment]
Oh, it looks like @KeyMapping and @JSON are just mutually exclusive

**Comment by F0x1d** [comment]
However, even without @KeyMapping @JSON is still not working properly

**Comment by joshuawright11** [comment]
Thanks @F0x1d - looks like this is a multiline issue. Somehow SwiftSyntax isn't picking up the arguments [here](https://github.com/joshuawright11/papyrus/blob/main/PapyrusPlugin/Sources/APIAttribute.swift#L68-L71) when the `@JSON(...)` is spaced on multiple lines.

If you space it on one line, it'll work (see [the test](https://github.com/joshuawright11/papyrus/commit/8f49dec80dab079cbe90bea28677b2167fdc5e48) I just added). I'll investigate how to see the arguments on separate lines.

**Comment by joshuawright11** [comment]
Okay, found the issue. Fixed in [v0.6.7](https://github.com/joshuawright11/papyrus/releases/tag/v0.6.7). 

Thank you!
